sourcegraph/about
# Add onboarding use cases page

Closes #5196 by adding an onboarding use cases page.

## To Test
- Check out the `/use-cases/onboarding` page and make sure that the copy and designs match the spec.
- ^^ on mobile.

**NB**: The design calls for the use of the `QuoteCarousel`, but that is not accurate: there is only one quote in that section, and it should not use the carousel.

Other discrepancies with the design spec:
- Font weights match what they are in our existing use case template (e.g. the headline with the `ul` and "How Sourcegraph helps" have the same font as the existing page template, rather than the design (assuming we want the pages to match)).
- The order of the customer logos section and CTA follow the desktop design spec rather than mobile (not sure why these were reversed in mobile?)
- The bolding in the "Developer onboarding is slow and expensive" sections is out of sync between mobile and desktop designs. I followed desktop.
- We are using Material Design icons here, since this is the icon library we use across our properties. The designs use [Phosphor](https://phosphoricons.com/), but CPT has determined that we will not be using icons from this library until we are ready to make a site-wide change. Until that point, we will be using a single icon library for the sake of consistency.
